safe-settings/suborgs.yaml/rulesets.yaml

```diff
@@ -0,0 +1,19 @@
+# https://github.com/github/safe-settings/blob/main-enterprise/docs/sample-settings/suborg.yml
```

Comment:
The downside of the app is things are mutually exclusive. If we are to enable this, it would disable any current `rulesets`. I've left it here for demonstration purposes.